### ISSUE TITLE
java.io.NotSerializableException for Standardizer

### ISSUE THREAD

**Issue by wunderbarr** [haifengl/smile#483]
Hello,
I want to save the standardizer model. But it throws the **_java.io.NotSerializableException_**.
## Code snippet
```
java.io.NotSerializableException: smile.feature.Standardizer
	at java.io.ObjectOutputStream.writeObject0(ObjectOutputStream.java:1184)
	at java.io.ObjectOutputStream.writeObject(ObjectOutputStream.java:348)
```

## Information

- What Java (OpenJDK, Orack JDK, etc.) are you using and which Java version
I use openjdk version "1.8.0_222".
- Which Smile version
I use 1.5.3 version.
- What is your build system (e.g. Ubuntu, MacOS, Windows, Debian )
I use Ubuntu16.04.

Thank you very much!

**Comment by haifengl** [comment]
It is fixed in 2.0.

**Comment by wunderbarr** [comment]
> It is fixed in 2.0.

Then how could I use the 2.0 pre-released version? Thank you!

**Comment by haifengl** [comment]
https://github.com/haifengl/smile/releases
